plugins/search-react/src/components/SearchAutocomplete/SearchAutocomplete.tsx

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ChangeEvent, ReactNode, useCallback, useMemo } from 'react';
+
+import {
+  CircularProgress,
+  ListItemIcon,
+  ListItemText,
+  ListItemTextProps,
+} from '@material-ui/core';
+import {
+  Value,
+  Autocomplete,
+  AutocompleteProps,
+  AutocompleteChangeDetails,
+  AutocompleteChangeReason,
+  AutocompleteRenderInputParams,
+} from '@material-ui/lab';
```

Comment:
I did a quick comparison, and it appears that only the "debug" prop has been removed from version 5.

Reply:
> Should we already remove the debug and rename the props?

hmm, good question, and it would be nice to prepare for it already now, but as we haven't migrated to v5 yet, I think we should keep them and I'd guess we will continue to support both for some time!